tests/commands/test_unset.py

```diff
@@ -8,32 +8,31 @@
          'lap top_ap ple_mac book pro']
 
 directories = ['.',
-               'simple',
                's p a c e s',
-               's p a/c e s',
                'r/e/c/u/r/s/i/v/e',
                'overlap/one',
                'overlap/two',
-               'very/very/very/deep']
+               'very/very/very/deep',
+               ]
 
 assets = [f"{d}/{f}.{i}" for f in files for i, d in enumerate(directories)]
 
+content_dict = {"key": "value",
```

Comment:
As a bias: I would like if the keys and values were predictable and iterated (e.g. `one`, `two`, `three`, etc).

It will make using in tests and debugging more obvious.

Reply:
This was done and will be pushed in a minute, thanks!